packages/info/lib/diagnoses/duplicates.js

```diff
@@ -3,25 +3,24 @@
 const chalk = require('chalk');
 const detectDuplicates = require('duplitect');
 
-module.exports = class DuplicatePackagesDiagnosisMixins {
+module.exports = class DuplicatePackagesMixins {
```

Comment:
Since it's a naming-related PR I'll go nitpicky 😄. `DuplicatePackagesMixins` sounds like we're duplicating stuff while here we want to **detect** when something is duplicated. Shall we use somehow `detect`?

Reply:
done